react/tic-tac-toe/src/app/screens/Home/screens/Matches/index.js

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+function Matchesitos(props) {
```

Comment:
What's this?

Reply:
he was a new component, but he was deleted.

Reply:
It*